optimum/exporters/openvino/__main__.py

```diff
@@ -328,6 +328,7 @@ def main_export(
             if dtype in [torch.float16, torch.bfloat16]:
                 patch_16bit = True
             loading_kwargs["torch_dtype"] = dtype
+        #           loading_kwargs["low_cpu_mem_usage"] = True
```

Comment:
```suggestion
```

Reply:
why ? I thought this was added specifically to reduce mem during export

Reply:
so `low_cpu_mem_usage` was removed in latest transformers version, but removing it will impact previous versions performance

Reply:
it was just added as part of experiment, somewhere between 4.51.0 and 4.51.3 transformers were broken and model loading required 2x by memory